spacy/util.py

```diff
@@ -394,13 +393,17 @@ def get_module_path(module: ModuleType) -> Path:
     return file_path.parent
 
 
+# Default value for passed enable/disable values.
+DEFAULT_PIPES_STATUS = SimpleFrozenList()
```

Comment:
I know we import it in the `language` module, but nevertheless I think this can be "private", i.e. `_DEFAULT_PIPES_STATUS`

And actually, maybe singular? `_DEFAULT_PIPE_STATUS`

Reply:
Renamed to `_DEFAULT_EMPTY_PIPES`. Not super happy with any of the names so far (`DEFAULT_PIPE_STATUS` included), but couldn't come up with a better one yet. Since this is private now and limited in its use anyway, I think it's alright if we go ahead and change the name later if we feel the need to?